code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -104,5 +104,11 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/storage/secure/briefcase = 3,
 	/obj/item/storage/toolbox/artistic = 2,
 	/obj/item/toy/eightball = 1,
+	/obj/item/gun/ballistic/automatic/toy = 2,
```

Comment:
These surprisingly don't have firing pins started in them I believe, use /obj/item/gun/ballistic/automatic/toy/unrestricted
from https://github.com/tgstation/tgstation/blob/057aa31cda6eb47505bc9fdd263fd1427b1276ed/code/modules/projectiles/guns/ballistic/toy.dm#L17

Reply:
the thing is, they do have firing pins, the unrestricted versions of non syndicate foam force are fucking pointless as they have the same firing pin
going to remove the pointless ones